sorting/README.md

```
@@ -0,0 +1,35 @@
+- **Space Complexity**: Some algorithms are in-place, others require extra memory.
+
+## Common Problems
+
+- Merge Sort
+- Quick Sort
+- Bubble Sort
+- Insertion Sort
+- Selection Sort
+
+## Time & Space Complexities of Merge Sort
+| Operation	    | Complexity   |
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 12

(MD010, no-hard-tabs)